chapters/hpmor-chapter-001.tex

```diff
@@ -169,3 +180,5 @@ \chapter{A Day of Very Low Probability}
 There was a long silence in the garden.
 
 Then a boy’s voice said, calmly and quietly, “What.”
+
```

Comment:
We shouldn't have any editor-specific settings, there's a project wordlist for this (anyway, this entry is empty!)

Reply:
ok, removed